src/views/Profile.vue

```diff
@@ -1,32 +1,64 @@
 <template>
-  <div id="profile_page" :class="this.$store.state.css">
-    <h1>Profil utilisateur</h1>
-    <h3>Cette page est encore en construction</h3>
-    <p>Si vous avez été redirigé automatiquement ici, c'est que votre abonnement n'est pas à jour</p>
-    <p><a class="teamSelector" v-on:click="goToBillingPortal">{{$t('BILLING_MANAGEMENT')}}</a></p>
-    <NavMenu :page="'profile'" />
+  <div :class="this.$store.state.css">
+    <div class="profilpage">
+      <div class="col-12 offset-0 col-sm-8 offset-sm-2 col-md-6 offset-md-3 col-lg-4 offset-lg-4 col-xxl-3 offset-xxl-5 ">
+        <div class="image-container">
+           <img id="logoface" alt="Vue logoface" src="../assets/logo_round.png" />
+        </div>
+         <h3 class="ProfilName">Nom Prenom</h3>
+        <li>
+            <ul>
+              <a class="buttonprofile" v-on:click="disabled"><div class="icone fas fa-credit-card"></div>{{
```

Comment:
Like that this button call a method called disabled so it doesn't work because this method doesn't exist but for user buttons seem clickable, maybe write disabled in class (an object can have several class' names) and add a particular CSS for these objects this a class disabled. (you can inspire from NavigationMenu.vue and its CSS ButtonMenu.less)
the goal is to make the application the more user-friendly possible and learn :)